### PR TITLE
[FEAT/INTRODUCTION] Create Introduction Page

### PR DESCRIPTION
**Pull request by baohuy2k3**
## Description

- Init Introduction page 

## Project

- [x] Website
- [ ] API
- [ ] Documentation
- [ ] Other... (please describe)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other... (please describe)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B
- [x] No tests needed

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Other... (please describe)

### CODE REVIEW

File: .eslintcache
Comment (foxminchan) [inline .eslintcache:1]: @baohuy2k3 

> **Warning**
> Remove and ignore this file

File: apps/website/src/features/Intro/components/ItemIntro.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Grid } from '@mui/material';
+import { itemIntroInfo } from '../intro.data';
+
+export default function ItemIntro() {
+  return (
+    <Grid
+      container
+      spacing={{ xs: 2, md: 3 }}
```
Comment (foxminchan) [inline apps/website/src/features/Intro/components/ItemIntro.tsx:8]: @baohuy2k3 Why don't you use tailwind class?

**Review by foxminchan — CHANGES_REQUESTED** [review]

File: apps/website/src/@types/images.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import denghi from '../assets/images/icons/intro/denghi.svg';
+import login from '../assets/images/icons/intro/login.svg';
+import payment from '../assets/images/icons/intro/payment.svg';
+import thucchien from '../assets/images/icons/intro/thuchien.svg';
+import timkiem from '../assets/images/icons/intro/timkiem.svg';
+import truyvan from '../assets/images/icons/intro/truyvan.svg';
+
+type IconIntroType= {
+    Share_Icon: string,
+    DanhGia_Icon: string,
```
Comment (qoucname2202) [inline apps/website/src/@types/images.ts:12]: @baohuy2k3 Convert all Vietnamese variables to English variables

File: apps/website/src/@types/images.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    Login_Icon: string,
+    Payment_Icon: string,
+    ThucHien_Icon: string,
+    TimKiem_Icon: string,
+    TruyVan_Icon: string,
+}
+
+export const IconIntro: IconIntroType ={
+    Share_Icon: share,
+    DanhGia_Icon: danhgia,
```
Comment (qoucname2202) [inline apps/website/src/@types/images.ts:23]: @baohuy2k3 Convert all Vietnamese variables to English variables

**Review by qoucname2202 — CHANGES_REQUESTED** [review]
@baohuy2k3 Fix code style for each file has comment

**Review by qoucname2202 — APPROVED** [review]
@foxminchan approved merger

**Review by foxminchan — APPROVED** [review]

**Review by foxminchan — CHANGES_REQUESTED** [review]
@baohuy2k3 fix code smell in sonarcloud

**Review by foxminchan — APPROVED** [review]

**Comment by foxminchan** [comment]
@baohuy2k3 merged

**Pull request merged**